tests/aws/services/lambda_/test_lambda_common.py

```diff
@@ -47,7 +47,7 @@ def snapshot_transformers(snapshot):
 
 
 @pytest.mark.skipif(
-    condition=get_arch() != "x86_64", reason="build process doesn't support arm64 right now"
+    condition=get_arch() != "amd64", reason="build process doesn't support arm64 right now"
```

Comment:
I recently talked to @dfangl that we should finally fix the platform detection properly. I'm not sure whether that skip was/is related to the CI infrastructure because I am able to build and run these tests on ARM (after uncommenting that annoying skip ;)